src/lib/items/Item.js

```diff
@@ -248,18 +249,31 @@ export default class Item extends Component {
           let dragGroupDelta = this.dragGroupDelta(e)
 
           if (this.props.moveResizeValidator) {
-            dragTime = this.props.moveResizeValidator(
+            const validResult = this.props.moveResizeValidator(
               'move',
               this.props.item,
-              dragTime
+              dragTime,
+              undefined,
+              this.props.order + dragGroupDelta,
+              this.props.order
             )
+            
+            if (validResult.time) {
+              dragTime = validResult.time
+            } else {
+              dragTime = validResult
+            }
+            if (validResult.newGroupIndex) {

```

Comment:
If newGroupIndex is set to 0, this statement will return false.
Should be more like:
`if (validResult.newGroupIndex !== undefined)`